contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/BasicFetchStrategy.java

```diff
@@ -89,82 +84,86 @@ private CheckSum copyFileWithCheckSum(HdfsFile source, File dest, CheckSumType c
         OutputStream output = null;
         long startTimeMS = System.currentTimeMillis();
 
-        for (int attempt = 0; attempt < fetcher.getMaxAttempts(); attempt++) {
```

Comment:
I agree with your preference, it's prettier with a space. However, the culprit is not my editor ;) ...

The original code was "the ugly format", which you changed to "the pretty format" as part of your code move in the ```FetchStrategy``` refactoring here:

https://github.com/voldemort/voldemort/commit/d70ed85e51620620b7ad2f21ca71f5e3537d01e8#diff-8645ea9a3775f1da51779f7d7d12d8e2L489

When I rebased my code on top of master, I copied the whole body of the function rather than editing each line one by one, so I brought back the old formatting, which I (and my editor) hadn't changed.

Anyway, I'll remove that change to keep your new format before merging into master.

-F